src/arviz_plots/plots/forestplot.py

```diff
@@ -176,30 +163,7 @@ def plot_forest(
         >>>     shade_label="school",
         >>> )
 
-    Extend the forest plot with an extra :term:`plot` with ess estimates.
-    To achieve that, we manually add a "column" dimension with size 3.
-    ``plot_forest`` only plots on the "labels" and "forest" coordinate values,
-    leaving the "ess" coordinate empty. Afterwards, we manually use
-    :meth:`.PlotCollection.map` with the ess result as data on the "ess" column
-    to plot their values.
-
-    .. plot::
-        :context: close-figs
-
-        >>> from arviz_plots import visuals
-        >>> import arviz_stats  # make accessor available
-        >>>
-        >>> c_aux = centered["posterior"].expand_dims(
-        >>>     column=3
-        >>> ).assign_coords(column=["labels", "forest", "ess"])
-        >>> pc = plot_forest(c_aux, combined=True)
-        >>> pc.map(
-        >>>     visuals.scatter_x, "ess", data=centered.azstats.ess().ds,
-        >>>     coords={"column": "ess"}, color="C0"
-        >>> )
-
-    Note that we are using the same :class:`~.PlotCollection`, so when using
-    ``map`` all the same aesthetic mappings used by ``plot_forest`` are used.
+    .. minigallery:: plot_forest
```

Comment:
```suggestion
     For a general introduction to batteries-included functions like this one and common
    usage examples see :ref:`plots_intro`. 
    .. minigallery:: plot_forest
```

Reply:
Left a comment above, but putting these two together might give users the idea they are related resources when they are completely different. One is quite general about functions inside `arviz_plots.plots` and covers the genral logic for plot_kwargs, aes_map, a bit on how to handle the returned object. Whereas the other are specific examples using the function. 

IMO, people who are beginners and want to learn to use arviz-plots should go to the first resource, people who already have an idea on how to use it but are searching for a specific plot or want to see what do some of the arguments actually do should go to the 2nd